docs/about/contributing.md

```diff
@@ -41,148 +41,158 @@ We use [semantic-release](https://www.npmjs.com/package/semantic-release), which
+### Source Code Management
 
-* **[scm-base][scm-base-repo]**: Common interface [![Version][scm-base-npm-image]][scm-base-npm-url]
-* **[scm-bitbucket][scm-bitbucket-repo]**: Bitbucket implementation [![Version][scm-bitbucket-npm-image]][scm-bitbucket-npm-url]
-* **[scm-github][scm-github-repo]**: Github implementation [![Version][scm-github-npm-image]][scm-github-npm-url]
+An SCM implementation is used as the interface between the API and an SCM. There are several implementations written in node around a common interface.
 
-### [Config Parser][config-parser-repo] [![Version][config-parser-npm-image]][config-parser-npm-url]
+* **[scm-base][scm-base-repo]**: Common interface
+* **[scm-bitbucket][scm-bitbucket-repo]**: Bitbucket implementation
+* **[scm-github][scm-github-repo]**: Github implementation
+
+### Templates
+
+Templates are snippets of predefined code that people can use to replace a job definition in their `screwdriver.yaml`. A template contains a series of predefined steps along with a selected Docker image.
+
+* **[templates][templates-repo]**: A repo for all build templates
+* **[template-main][template-main-repo]**: The CLI for validating and publishing job templates
+* **[template-validator][template-validator-repo]**: A tool used by the API to validate a job template
+
+### [Config Parser][config-parser-repo]
 
 Node module for validating and parsing user's `screwdriver.yaml` configurations.
 
-### [Guide][guide-repo]
+### [Guide][guide-repo] & [Homepage][homepage-repo]
 
-This documentation! Everything you ever hoped to know about the Screwdriver project.
+The [Guide][guide-repo] is documentation! Everything you ever hoped to know about the Screwdriver project.
+The [Homepage][homepage-repo] is the basic one-pager that powers [Screwdriver.cd][homepage].
+
+### [UI][ui-repo]
+
+The Ember-based user interface of Screwdriver.
 
 ### Miscellaneous Tools
 
+* **[circuit-fuses][circuit-fuses-repo]**: Wrapper to provide a node-circuitbreaker with callback interface
 * **[client][client-repo]**: Simple Go-based CLI for accessing the Screwdriver API
 * **[gitversion][gitversion-repo]**: Go-based tool for updating git tags on a repo for a new version number
-* **[circuit-fuses][circuit-fuses-repo]**: Wrapper to provide a node-circuitbreaker w/ callback interface [![Version][circuit-fuses-npm-image]][circuit-fuses-npm-url]
-* **[keymbinatorial][keymbinatorial-repo]**: Generates the unique combinations of key values by taking a single value from each keys array [![Version][keymbinatorial-npm-image]][keymbinatorial-npm-url]
+* **[keymbinatorial][keymbinatorial-repo]**: Generates the unique combinations of key values by taking a single value from each keys array
+* **[toolbox][toolbox-repo]**: Repository for handy Screwdriver-related scripts and other tools
+* **[hashr][hashr-repo]**: Wrapper module for generating ids based on a hash
```

Comment:
Sure :) I'll do it for dynamic-dynamodb, datastore-dynamodb, client-repo, and hashr.

If there are others that aren't used anymore, someone who has actually worked on them and knows the reasons for their out-of-dateness should probably take the initiative to document that.